### PR TITLE
Get event lambda

### PR DESCRIPTION
**Pull request by ethanmills**
Get an event from the datashare API and validate the fields against the LEV API. I intend to add CW metrics and dashboarding for this, but that will come later

### CODE REVIEW

File: terraform/modules/lambdas/common.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import json
```
Comment (ethanmills) [inline terraform/modules/lambdas/common.py:1]: I've done a bit of commonisation here. Open to being told this is a terrible idea

File: terraform/modules/len/lambdas/len.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 import math
 import os
-from random import randint
-from urllib import request, parse
-import json
 from datetime import datetime
+from random import randint
+from urllib import request
+
+from common import get_auth_token
```
Comment (ethanmills) [inline terraform/modules/len/lambdas/len.py:9]: Sorry, this auto optimize on save

File: terraform/modules/consumer/lambdas/get_event.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+gdx_url = os.environ["gdx_url"]
+events_url = gdx_url + "/events"
+
+auth_url = os.environ["auth_url"]
+
+client_id = os.environ["client_id"]
+client_secret = os.environ["client_secret"]
+
+lev_api_url = os.environ["lev_api_url"]
```
Comment (olevett) [inline terraform/modules/consumer/lambdas/get_event.py:21]: I find this surprising, I'd have assumed that our event consumption is ignorant of anything but GDX, are we doing some validation of things against LEV?

Comment (olevett) [reply terraform/modules/consumer/lambdas/get_event.py:21]: I scrolled a bit further on and found the answer. I think this is worth an explanatory note though, in that normal consumers would not need to do this.

**Review by olevett — APPROVED** [review]

**Pull request merged**